crates/cargo-test-macro/build.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
+    println!("cargo:rerun-if-changed=tests/snapshots");
```

Comment:
We want to rebuild the snapshot tests when a new test is added.  These snapshot tests live in the `cargo` crate but this forcing the `cargo-test-macro` crate to rebuild which is a lot more rebuilds than is needed.

Reply:
I'll move it

Reply:
Just to confirm I should move it to cargo's `build.rs`?

Reply:
Yes